doc/Command_Index_gregorio.tex

```diff
@@ -1080,6 +1104,18 @@ \section{Gregorio Controls}
   \#5 & string & the line, byte offset, and column address for textedit links when point-and-click is enabled\\
 \end{argtable}
 
+\macroname{\textbackslash GreSharpSoft}{\#1\#2\#3\#4\#5}{gregoriotex-signs.tex}
+Macro to typeset a soft sharp.
+
+\begin{argtable}
+  \#1 & integer & Height number of the sharp.\\
+  \#2 & \texttt{0} & The sharp is not part of the clef.\\
+  & \texttt{1} & The sharp is part of the clef (doesn't happen).\\
+  \#3 & \TeX\ code & signs to typeset before the glyph (typically additional bars, as they must be "behind" the glyph)\\
```

Comment:
And a third time.

Reply:
This was a copy-paste from the existing documentation for `\GreFlat` and friends. I've changed them throughout the whole manual.